### DISCUSSION TITLE
Music Assistant

### DISCUSSION THREAD

**Discussion by altrdev** [osk2/yamaha-soundbar#18] [General]
Hi, I have this issue with Music Assistant. The song progress bar doesn't move forward, so Music Assistant doesn't know the song has ended. As a result, instead of moving to the next one, it restarts the same song. This doesn't happen with other media players. Do you have any ideas about it?

**Comment by osk2** [comment]
I have no experience with Music Assistant, but it would be a bug if the media player entity behaved the same way. Can you confirm whether player progress works on the media player entity?
